definitions/contested/json/CaseField/CaseField.json

```diff
@@ -8185,5 +8185,77 @@
     "FieldType": "YesOrNo",
     "SecurityClassification": "Public",
     "Searchable": "N"
+  },
+  {
+    "LiveFrom": "17/02/2025",
+    "CaseTypeID": "FinancialRemedyContested",
+    "ID": "expressCaseParticipation",
+    "Label": " ",
+    "FieldType": "Text",
+    "SecurityClassification": "Public",
+    "Searchable": "Y"
+  },
+  {
+    "LiveFrom": "07/03/2025",
+    "CaseTypeID": "FinancialRemedyContested",
+    "ID": "suitableForExpressCaseLabel",
+    "Label": "You have been automatically entered into the 'express financial remedy procedure' pilot. This is because you have noted that you and the respondent's estimated total combined net assets are £250,000 or less, and your case has been allocated to one of the participating courts.\r\n\r\nYou can find out more information on the pilot procedure here:\r\n<a href=\"https://www.gov.uk/guidance/what-to-expect-if-you-are-in-the-express-financial-remedy-pilot\">https://www.gov.uk/guidance/what-to-expect-if-you-are-in-the-express-financial-remedy-pilot/</a>.",
+    "FieldType": "Label",
+    "SecurityClassification": "Public",
+    "Searchable": "N"
+  },
```

Comment:
Just FYI, if resolving conflicts with this one, check the <a> tag.  Have added some data to open in a new tab.